tap_square/sync.py

```diff
@@ -33,7 +33,31 @@ def sync(config, state, catalog):
             start_time = singer.get_bookmark(state, tap_stream_id, replication_key, config['start_date'])
             bookmarked_cursor = singer.get_bookmark(state, tap_stream_id, 'cursor')
 
-            if stream_obj.replication_method == 'INCREMENTAL':
+            if tap_stream_id == 'shifts':
```

Comment:
Maybe we should make this some other flag on the stream instead of hardcoding to the stream id 'shifts'?